ui/src/main/webapp/src/app/project/migration-project-form.component.ts

```diff
@@ -55,6 +56,9 @@ export class MigrationProjectFormComponent extends FormComponent implements OnIn
             }
 
             this.isInWizard = flatRouteData.data.hasOwnProperty('wizard') && flatRouteData.data['wizard'];
+
+            if(!this.editMode) // Creating a new project.
+                this._migrationProjectService.deleteProvisionalProjects().subscribe(res => {});
```

Comment:
Lol, that's what happens when I comment at night on a weekend, I guess. The commit that I sent was unrelated. I'm not really sure why I moved it, tbh. Just too many changes in a hurry, I think.

If you want, I can take you through the issues on Monday. There were some significant functional issues and I think I can grab it from your original state easily enough.

Having said that, most of it was good and I'm glad that the feature will make it in. I'll try to move the offending line before the final build.